### PR TITLE
Generate reports dir automatically

### PR DESCRIPTION
**Pull request by wendigo**
Follow-up to https://github.com/prestosql/presto/pull/4731#discussion_r467504845

### CODE REVIEW

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java [hunk 1]
```diff
@@ -297,6 +297,11 @@ private void awaitTestsCompletion(Environment environment)
                 throw new RuntimeException("Interrupted", e);
             }
         }
+
+        private static Path createReportsDir(String reportsDir, String environment)
```
Comment (losipiuk) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:301]: rename to `buildFinalReportsDir`

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java [hunk 2]
```diff
@@ -98,7 +100,7 @@ public void run()
         public String environment;
 
         @Option(name = "--reports-dir", title = "reports dir", description = "location of the reports directory")
-        public String reportsDir;
+        public String reportsDir = "presto-product-tests/target/reports/";
```
Comment (losipiuk) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:103]: raname to `reportsDirBase`?

**Comment by losipiuk** [comment]
@wendigo when automation completes please check if reports landed when they should.

**Comment by wendigo** [comment]
@losipiuk Log contains valid dirs. As we are not publishing artifacts I cannot see those files (yet) but locally files are written where they should be.

**Pull request merged**